Makefile

```
@@ -24,9 +25,16 @@ etcd-release: ## Interactively starts the release workflow.
 ##@ Helm dev targets - The etcd needs a copy of the targets from k8s.mk without image-import because we use a external image here.
 
 .PHONY: k8s-helm-etcd-apply
```

```suggestion
.PHONY: helm-etcd-apply
```
We could rename the targets to match the new naming-pattern from k8s-component.mk